class.jetpack.php

```diff
@@ -52,8 +52,8 @@ class Jetpack {
 	);
 
 	var $plugins_to_deactivate = array(
-		'stats'               => array( 'stats/stats.php', 'WordPress.com Stats' ),
-		'shortlinks'          => array( 'stats/stats.php', 'WordPress.com Stats' ),
+		'stats'               => array( 'stats/stats.php', 'Site Stats' ),
```

Comment:
Fixed in https://github.com/Automattic/jetpack/commit/fd650dfc0515ecbc22965c939c95042b87042105